adage/analyze/api.py

```diff
@@ -342,7 +350,7 @@ def apply_filters(self, request, applicable_filters):
 
 class ParticipationResource(ModelResource):
     node = fields.IntegerField(attribute='node_id', null=False)
```

Comment:
Performance is an excellent reason. I also agree wholeheartedly with the rationale and the suggestion to comment about deliberately not using `ForeignKey`.